core/base/document.php

```diff
@@ -1258,4 +1259,14 @@ protected function get_post_statuses() {
 	protected function get_have_a_look_url() {
 		return $this->get_permalink();
 	}
+
+	public function get_name() {
+		return static::get_type();
+	}
+
+	public static function get_create_url() {
+		$base_create_url = Utils::get_create_new_post_url( Source_Local::CPT );
```

Comment:
Move from utils to the documents manager.

Reply:
Handled.